Library/Homebrew/cmd/vendor-install.sh

```diff
@@ -20,6 +20,11 @@ then
 elif [[ -n "$HOMEBREW_LINUX" ]]
 then
   case "$HOMEBREW_PROCESSOR" in
+    armv[6789]*)
+      ruby_URL="https://homebrew.bintray.com/bottles-portable-ruby/portable-ruby-2.3.3_2.armv6_linux.bottle.tar.gz"
+      ruby_URL2="https://github.com/Homebrew/homebrew-portable-ruby/releases/download/2.3.3_2/portable-ruby-2.3.3.armv6_linux.bottle.tar.gz"
```

Comment:
`portable-ruby-2.3.3.armv6_linux.bottle.tar.gz` is not the correct file name. This is incorrect (and my fault) on `x86_64` below too; could you fix that as well?

Reply:
Done.